packages/orbit-components/src/Slider/index.tsx

```diff
@@ -410,13 +354,20 @@ const PureSlider = ({
   const hasHistogram = histogramLoading || !!histogramData;
 
   return (
-    <StyledSlider data-test={dataTest} id={id}>
+    <div className="relative" data-test={dataTest} id={id}>
       {hasHistogram ? (
         <>
           <Hide on={["smallMobile", "mediumMobile", "largeMobile"]} block>
             {renderSliderTexts(true)}
           </Hide>
-          <StyledSliderContent focused={focused}>
+          <div
+            // TODO use tb:-inset-x-lg once we don’t support ios_safari 14
```

Comment:
Do you mean `-start-lg`? `-inset-x-lg` applies normal left and right which are supported by safari 14.

Reply:
I kinda expected that `-inset-x-` produces `inset-inline`… Updated, thanks!